public/css/history.css

```diff
@@ -1,12 +1,23 @@
 .wrapper {
-	padding-top: 130px;
-	background-color: #fafafa;
-	color: #2b2b2b;
+	padding-top: 66px;
+}
+
+#history_logo {
+	position: absolute;
+	top: 170px;
+	left: 50%;
+	transform: translateX(-50%);
+	max-width: 100%;
+}
```

Comment:
On small screens, the logo butts up against the sides of the screen. Add some side padding/margin (30ish px) so this doesn't happen.